src/ucp/proto/proto_common.c

```diff
@@ -845,8 +845,8 @@ ucs_status_t ucp_proto_request_zcopy_id_reset(ucp_request_t *req)
 static void ucp_proto_stub_fatal_not_implemented(const char *func_name,
                                                  ucp_request_t *req)
 {
-    ucs_fatal("%s request %p proto %s, not implemented", func_name, req,
-              req->send.proto_config->proto->name);
+    ucs_fatal("'%s' is not implemented for protocol %s (req: %p)", func_name,
```

Comment:
Minor: `ucp_trace_req` has the following format: "req %p:  message text". Maybe it worth making this one consistent with `ucp_trace_req`?

Reply:
not sure, because this is a Fatal error message and not a regular log message